src/assets/img/iconArrowGrey.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
in future we should make it a task to go through and unify all our svgs into the @audius/stems :D